recipe/activate-clang.sh

```diff
@@ -195,6 +195,25 @@ else
     diff -U 0 -rN /tmp/old-env-$$.txt /tmp/new-env-$$.txt | tail -n +4 | grep "^-.*\|^+.*" | grep -v "CONDA_BACKUP_" | sort
     rm -f /tmp/old-env-$$.txt /tmp/new-env-$$.txt || true
   fi
+
+  # fix prompt for zsh
+  if [[ -n "${ZSH_NAME}" ]]; then
+    _conda_clang_precmd() {
+      CONDA_CLANG_OLDHOST="${HOST}"
+      HOST="${CONDA_BACKUP_HOST}"
+    }
+
+    [[ -z \$precmd_functions ]] && precmd_functions=()
+    precmd_functions=(\$precmd_functions _conda_clang_precmd)
+
+    _conda_clang_preexec() {
+      HOST="${CONDA_CLANG_OLDHOST}"
```

Comment:
Let's add a new variable `CONDA_TOOLCHAIN_HOST` and use it in our recipes and also do the following change.
```suggestion
      HOST="${CONDA_TOOLCHAIN_HOST}"
```

Reply:
Sure!

Reply:
done.